pyccel/ast/core.py

```diff
@@ -2271,6 +2210,61 @@ def prefix(self):
         """
         return self._prefix
 
+class ConstructorCall(DottedFunctionCall):
+
+    """
+    It represents a constructor for undefined function classes.
+
+    This class is used to create instances of undefined function classes, providing the ability to
```

Comment:
Do you understand what the word "undefined" is talking about here? I don't find it very clear, I assume it was copied from the original docstring? Unless you find it clear?

My best guess is that it is talking about class instances which are declared but not defined/initialised. The constructor then defines/initialises them. But if that is the case then the following sentence makes no sense:
> This class is used to create instances of undefined function classes,

Reply:
Fixed